pyclam/chaoda.py

```diff
@@ -88,19 +88,18 @@ def graph_neighborhood(graph: Graph, normalize: str = 'gaussian') -> np.array:
     """
     _catch_normalization_mode(normalize)
 
-    def _bft(start: Cluster) -> List[int]:
-        path_length: List[int] = list()
+    def _bft(start: Cluster, steps: int) -> int:
```

Comment:
seems odd to require a set amount of steps for bft

Reply:
it returns the number of clusters within a graph distance of 'steps' from 'start'. I changed the name and added a docstring